app/src/main/java/com/poupa/vinylmusicplayer/service/MusicService.java

```diff
@@ -619,6 +620,35 @@ public void onResourceReady(@NonNull Bitmap resource, Transition<? super Bitmap>
         }
     }
 
+    public void updateMediaSessionLyric(String line) {
+        final Song song = getCurrentSong();
+
+        if (song.id == -1) {
+            mediaSession.setMetadata(null);
+            return;
+        }
+
+        List<String> splitlist = StringUtil.wideCharacterSplit(line,20);
```

Comment:
what is 20 please?

Reply:
In car display, the space is limited to 20 letters, so I need to split the line as in many cases, the line is longer than 20 letters.